apps/prairielearn/src/pages/administratorQuery/administratorQuery.html.ts

```diff
@@ -0,0 +1,333 @@
+  error,
+  result,
+  sqlFilename,
+  info,
+}: {
+  resLocals: Record<string, any>;
+  has_query_run: boolean;
+  query_run_id: string | null;
+  formatted_date: string | null;
+  params: AdministratorQueryRunParams | null;
+  error: string | null;
+  result: AdministratorQueryResult | null;
+  sqlFilename: string;
+  info;
+}) {
+  function renderHeader(columns, col) {
+    const row = {};
+    columns.forEach((c) => {
+      row[c] = true;
+    });
+
+    if (col === 'course_id' && 'course' in row) {
+      null;
+    } else if (col === 'course_instance_id' && 'course_instance' in row) {
+      null;
+    } else if (col === 'assessment_id' && 'assessment' in row && 'course_instance_id' in row) {
+      null;
+    } else if (/^_sortval_/.test(col)) {
+      null;
+    } else {
+      return html`<th>${col}</th>`;
+    }
+  }
+
+  function render(row, col) {
+    let tdAttributes = '';
+    if ('_sortval_' + col in row) {
+      tdAttributes = `data-text="${row['_sortval_' + col]}"`;
+    }
+
+    if (col === 'course_id' && 'course' in row) {
+      null;
+    } else if (col === 'course' && 'course_id' in row) {
+      return html`
+        <td ${tdAttributes}>
+          <a href=${`${resLocals.urlPrefix}/course/${row['course_id']}`}>${row[col]}</a>
+        </td>
+      `;
+    } else if (col === 'course_instance_id' && 'course_instance' in row) {
+      null;
+    } else if (col === 'course_instance' && 'course_instance_id' in row) {
+      return html`
+        <td ${tdAttributes}>
+          <a href=${`${resLocals.urlPrefix}/course_instance/${row['course_instance_id']}`}
+            >${row[col]}</a
+          >
+        </td>
+      `;
+    } else if (col === 'assessment_id' && 'assessment' in row && 'course_instance_id' in row) {
+      null;
+    } else if (col === 'assessment' && 'assessment_id' in row && 'course_instance_id' in row) {
+      return html`
+        <td ${tdAttributes}>
+          <a
+            href=${`${resLocals.urlPrefix}/course_instance/${row['course_instance_id']}/instructor/assessment/${row['assessment_id']}`}
+            >${row[col]}</a
+          >
+        </td>
+      `;
+    } else if (/^_sortval_/.test(col)) {
+      null;
+    } else if (row[col] == null) {
+      return html` <td ${tdAttributes}></td> `;
+    } else if (info.resultFormats && info.resultFormats[col] === 'pre') {
+      return html`
+        <td ${tdAttributes}>
+          <pre>${row[col]}</pre>
+        </td>
+      `;
+    } else if (typeof row[col] == 'object') {
+      return html`
+        <td ${tdAttributes}>
+          <code>${JSON.stringify(row[col])}</code>
+        </td>
+      `;
+    } else {
+      return html` <td ${tdAttributes}>${row[col]}</td> `;
+    }
+  }
+
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head') %>", resLocals)}
+        <link
+          href=${resLocals.node_modules_asset_path('highlight.js/styles/default.css')}
```

Comment:
```suggestion
          href=${nodeModulesAssetPath('highlight.js/styles/default.css')}
```

You will need to import this from lib/assets. Same for other uses of this function.

Reply:
Resolved in commit 227f6c3.